code/_onclick/hud/alert.dm

```diff
@@ -515,20 +551,49 @@ so as to remain in compliance with the most up-to-date laws."
 /obj/screen/alert/notify_action/Click()
 	if(!usr || !usr.client)
 		return
-	if(!target)
-		return
 	var/mob/dead/observer/G = usr
 	if(!istype(G))
 		return
-	switch(action)
-		if(NOTIFY_ATTACK)
-			target.attack_ghost(G)
-		if(NOTIFY_JUMP)
-			var/turf/T = get_turf(target)
-			if(T && isturf(T))
-				G.loc = T
-		if(NOTIFY_FOLLOW)
-			G.ManualFollow(target)
+
+	if(poll)
+		if(poll.sign_up(G))
+			// Add a small overlay to indicate we've signed up
+			display_signed_up()
+	else if(target)
+		switch(action)
+			if(NOTIFY_ATTACK)
+				target.attack_ghost(G)
+			if(NOTIFY_JUMP)
+				var/turf/T = get_turf(target)
+				if(T && isturf(T))
+					G.loc = T
+			if(NOTIFY_FOLLOW)
+				G.ManualFollow(target)
+
+/obj/screen/alert/notify_action/proc/display_signed_up()
+	var/image/I = image('icons/mob/screen_gen.dmi', icon_state = "selector")
+	I.layer = FLOAT_LAYER
+	I.plane = FLOAT_PLANE + 2
+	overlays += I
+
+/obj/screen/alert/notify_action/proc/display_stacks(stacks = 1)
+	if(stacks <= 1)
+		return
+
+	var/obj/O = new
+	O.maptext = "<span style='font-weight: bold; font-size: 32px; color: yellow;'>[stacks]x</span>"
```

Comment:
Set the font family here to `Small Fonts` so it renders crisply with no anti-aliasing (The same font is used on status displays and now the ticket machine). Do this in all other places its used.